.github/workflows/cypress-custom.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [ 16 ]
+        node-version: [ 18 ]
```

Comment:
As discussed lets adapt https://github.com/nextcloud/.github/blob/master/workflow-templates/cypress.yml#L63-L76 to read them from package.json

Reply:
Oh good to know, thanks for sharing that